subsys/testsuite/ztest/src/ztest.c

```diff
@@ -373,10 +376,6 @@ static int run_test(struct unit_test *test)
 		run_test_functions(test);
 	}
 
-	phase = TEST_PHASE_TEARDOWN;
```

Comment:
This change is causing a CI build failure. In particular, to reproduce:

```
$ scripts/twister -v -p qemu_cortex_a53_smp -T tests/kernel/mem_protect/userspace/
```

Reply:
working on it